src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.resx

```diff
@@ -4560,4 +4560,7 @@
   <data name="SQL_SettingCredentialWithInteractive" xml:space="preserve">
     <value>Cannot set the Credential property if 'Authentication=Active Directory Interactive' has been specified in the connection string.</value>
   </data>
+  <data name="SqlConnection_ServerProcessId" xml:space="preserve">
```

Comment:
Missing item in SR.resx

Reply:
It's not desired actually, this is NetFx specific property description.